python/nvtx/colors.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import functools
+
+_NVTX_COLORS = {
+    "green": 0x008000,
+    "blue": 0x0000FF,
+    "yellow": 0xFFFF00,
+    "purple": 0x800080,
+    "rapids": 0x7400FF,
+    "cyan": 0x00FFFF,
+    "red": 0xFF0000,
+    "white": 0xFFFFFF,
+    "darkgreen": 0x006400,
+    "orange": 0xFFA500,
+}
+
+
+@functools.lru_cache()
+def color_to_hex(color=None):
+    """
+    Convert color to ARGB hex value.
+    """
+    if color is None:
+        color = "blue"
```

Comment:
Instead of this check we should just add a dictionary entry for `None` to have a default color.

Reply:
Good idea - done!